pylint/checkers/variables.py

```diff
@@ -590,6 +610,8 @@ def mark_as_consumed(self, name, consumed_nodes):
     def get_next_to_consume(self, node):
         """
         Return a list of the nodes that define `node` from this scope.
+        If it is uncertain whether a node will be consumed, such as statement in
```

Comment:
This and the docstring of `consumed_uncertain` are very clear. This part of the codebase will never be fully self explanatory but this really helps! Thanks a lot for all the iterations and working with me 😄 Much appreciated!